README.md

```diff
@@ -36,17 +36,45 @@ There are several ways to install this app onto a workload cluster.
 If you have access to the Kubernetes API on the management cluster, you could create
 the App CR and ConfigMap directly.
 
-Here is an example that would install the app to
+Here is an example that would install the bundle to
 workload cluster `abc12`:
 
 ```yaml
-# appCR.yaml
+# user-values-configmap.yaml
 
+apiVersion: v1
+data:
+  values: |
+    clusterID: abc12
+    organization: someOrg
+kind: ConfigMap
+metadata:
+  name: linkerd-bundle-user-values
+  namespace: abc12
 ```
 
 ```yaml
-# user-values-configmap.yaml
+# appCR.yaml
 
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app.kubernetes.io/name: linkerd-bundle
+    app-operator.giantswarm.io/version: 0.0.0
+  name: linkerd-bundle-abc12
+  namespace: abc12
+spec:
+  catalog: giantswarm-playground
+  kubeConfig:
+    inCluster: true
```

Comment:
This would install the app in the management cluster? Don't know from head right now, would need to look up the details, but probably can just judge if it's meant like this or an oversight ;)

Reply:
I guess this is the idea behind the app bundle? AppCRs are created on the management cluster

Reply:
Ok, then I need to fetch up on that. Could it be a generic thing then? A `bundle-app` with a list of apps and one values file to be splited across them? Maybe this is kind of the case here? ;D Thanks for the hint.

Reply:
Indeed, this config is mandatory for bundles, as it's installing a bunch of apps in the MC. 
It's all explained in [this document](https://docs.giantswarm.io/getting-started/app-platform/app-bundle). In particular the `Configuring the bundle’s App CR for the unique App Operator` section.

> Could it be a generic thing then? A bundle-app with a list of apps and one values file to be splited across them?

it is an option, but it might be tricky to deal with releases and default values.